src/index.css

```
@@ -1,8 +1,6 @@
 body {
 	margin: 0;
-	font-family:
-		-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell",
-		"Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	font-family: "Onest", system-ui, -apple-system, "Segoe UI", "Roboto", "Helvetica Neue", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Onest" (font-family-name-quotes)

(font-family-name-quotes)